extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -359,6 +359,40 @@ describes.realWin('amp-story', {
           return expect(replaceStub).to.not.have.been.called;
         });
   });
+  describe('amp-story continue anyway', () => {
+
+    it('should not display layout', () => {
+      AmpStory.isBrowserSupported = () => false;
+      story = new AmpStory(element);
+      const dispatchStub = sandbox.stub(story.storeService_, 'dispatch');
+      createPages(story.element, 2, ['cover', 'page-4']);
+      story.buildCallback();
+      return story.layoutCallback()
+          .then(() => {
+            expect(dispatchStub).to.have.been.calledWith(
+                Action.TOGGLE_SUPPORTED_BROWSER, false
+            );
+          });
+    });
+
+    it('should render the amp story', () => {
```

Comment:
Our unit test should emulate user behavior i.e. click event on the "continue anyway" button.

Reply:
We discussed this in person, and I'll implement what we talked about!